code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -79,6 +85,44 @@
 				return 0
 	return 1
 
+/obj/item/storage/backpack/attackby(atom/A, mob/user, click_parameters)
```

Comment:
Respect the parent function header
`attackby(obj/item/attacking_item, mob/user, params)`